apisix/plugins/ldap-auth.lua

```diff
@@ -131,7 +131,8 @@ function _M.rewrite(conf, ctx)
 
     local user, err = extract_auth_header(auth_header)
     if err then
-        return 401, { message = err }
+        core.log.warn(err)
+        return 401, { message = "Invalid user authorization" }
```

Comment:
```suggestion
        return 401, { message = "Invalid authorization in request" }
```
would be more match the "Missing authorization in request"?